### PR TITLE
Add 1.20 & 1.20.1 Support.

### PR DESCRIPTION
**Pull request by Marcobohl**
- Change Spigot version 
- Change FastAsyncWorldEdit version 
- Change plotsquared version

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,14 +57,18 @@
             <id>intellectualsites snapshots</id>
             <url>https://mvn.intellectualsites.com/content/repositories/snapshots/</url>
         </repository>
+        <repository>
+            <id>papermc</id>
+            <url>https://repo.papermc.io/repository/maven-public/</url>
+        </repository>
```
Comment (Nononitas) [inline pom.xml:63]: Seems unnecessary as the paper api isn't used?

File: src/main/java/de/nononitas/plotborder/PlotBorder.java [hunk 2]
```diff
@@ -90,7 +90,7 @@ public void onEnable() {
             return;
         }
         String plotVersion = Bukkit.getPluginManager().getPlugin("PlotSquared").getDescription().getVersion();
-        if (!plotVersion.startsWith("6.")) {
+        if (!plotVersion.startsWith("7.")) {
```
Comment (Nononitas) [inline src/main/java/de/nononitas/plotborder/PlotBorder.java:93]: If there are no API changes that needs to be implemented why not support v6 and v7?

Comment (Marcobohl) [reply src/main/java/de/nononitas/plotborder/PlotBorder.java:93]: You are right, I will test it

File: pom.xml (hunk 1 above)
Comment (Marcobohl) [reply pom.xml]: I have removed the papermc repository.

File: src/main/java/de/nononitas/plotborder/PlotBorder.java (hunk 2 above)
Comment (Marcobohl) [reply src/main/java/de/nononitas/plotborder/PlotBorder.java]: I have added Support back for 6. it works with the Version 6 fine.

**Pull request merged**